src/test/java/io/confluent/examples/streams/GenericAvroIntegrationTest.java

```diff
@@ -51,18 +49,23 @@
  */
 public class GenericAvroIntegrationTest {
 
+  // A mocked schema registry for our serdes to use
+  private static final String SCHEMA_REGISTRY_SCOPE = GenericAvroIntegrationTest.class.getName();
```

Comment:
Instead of requiring test authors to create their own scopes by creating their own static client references, we are now just letting them give each registry a named scope. They can use the same name everywhere (and therefore get the same registry everywhere), or give every test its own uniquely named scope (which I'm demonstrating).